src/containers/JudgingPanel.js

```diff
@@ -108,24 +108,30 @@ const calculateResiduals = project => {
 
 const getProjectData = async () => {
   const projectDocs = await projectsRef.get()
-  return projectDocs.docs.map(projectDoc => projectDoc.data())
+  return projectDocs.docs.map(projectDoc => {
+    return { ...projectDoc.data(), id: projectDoc.id }
+  })
 }
 
 const getGrades = async () => {
-  const gradeData = (await getProjectData()).reduce((acc, project) => {
+  const gradeData = []
+  const projectData = await getProjectData()
+  projectData.forEach(project => {
```

Comment:
would filter make more sense here

Reply:
not sure because filter takes a predicate so id have to do a filter then a foreach?